gitops/snapshot.go

```diff
@@ -137,8 +137,8 @@ const (
 	// the global candidate list.
 	SnapshotAddedToGlobalCandidateListCondition = "AddedToGlobalCandidateList"
 
-	// AppStudioTestSucceededConditionPassed is the reason that's set when the AppStudio tests succeed.
-	AppStudioTestSucceededConditionPassed = "Passed"
+	// AppStudioTestSucceededConditionSatisfied is the reason that's set when the AppStudio tests succeed.
+	AppStudioTestSucceededConditionSatisfied = "Passed"
```

Comment:
I don't think this name change really makes sense.  Is there any way to make  Snyk ignore all instances of `AppStudioTestSucceededConditionPassed`?  I know some tools have a line annotation option to manually override certain errors

Reply:
Afaik, there's no way to ignore a specific instance of rule violation in snyk, only ignoring the entire file or ignoring the entire rule, none of which would be good practice imo.

That leaves renaming the constant as the best option, and though I agree it's tough to have a satisfactory replacement for `Passed`, the current one seems like a good compromise to me.